.mypy.ini

```diff
@@ -3,7 +3,7 @@ warn_return_any = False
 warn_unused_configs = True
 mypy_path = src/python/deepgnn
 
-files=src/python/deepgnn
+files=src/python/deepgnn,examples/*/*/*.py
```

Comment:
nit: use globbing to much every file in the repo `**/*.py`

Reply:
I've updated it to "src/python/deepgnn,examples" instead, globbing is hard since bazel need to be ignored.

Reply:
use [exclude section](https://mypy.readthedocs.io/en/stable/config_file.html#confval-exclude)

Reply:
I spent quite a while trying to get that to work and it seems to have weird behavior and I cannot get it to work. I think the order of the commands matter and maybe there are some overrides between commands or something that is not documented.

Reply:
Create an issue on the mypy github page and describe what you tried to do?

Reply:
Right now files=src/python/deepgnn,examples which is not so bad. It is probably not worth the effort?

Reply:
Well, you forgot to add `docs` folder, so it seems worth to use a correct pattern and exclude only bazel-bin folder.

Reply:
docs does not have any code to type check. I dont think we should ever have code outside of src and examples to need a complex pattern.